### PR TITLE
Small improvements for local discord bot

### PR DESCRIPTION
**Pull request by sz-piotr**

### CODE REVIEW

File: packages/backend/src/core/DiscoveryWatcher.ts [hunk 1]
```diff
@@ -209,11 +215,20 @@ export class DiscoveryWatcher {
...
     }
 
-    for (const message of messages) {
-      await this.discordClient.sendMessage(message, channel).then(
-        () => this.logger.info('Notification to Discord has been sent'),
-        (e) => this.logger.error(e),
-      )
+    const arrayMessages = Array.isArray(messages) ? messages : [messages]
+    for (const message of arrayMessages) {
+      type Channel = 'PUBLIC' | 'INTERNAL'
```
Comment (michalsidzej) [inline packages/backend/src/core/DiscoveryWatcher.ts:220]: That's the first time i see a type definition inside the function. You sure you want to leave it here?

File: packages/backend/src/core/DiscoveryWatcher.ts [hunk 2]
```diff
@@ -209,11 +215,20 @@ export class DiscoveryWatcher {
...
+      const channels: Channel[] = options?.internalOnly
+        ? ['INTERNAL']
+        : ['PUBLIC', 'INTERNAL']
+
+      for (const channel of channels) {
+        console.log(channels)
```
Comment (michalsidzej) [inline packages/backend/src/core/DiscoveryWatcher.ts:226]: Hello

File: packages/backend/src/peripherals/discord/DiscordClient.ts [hunk 3]
```diff
@@ -13,18 +13,20 @@ export class DiscordClient {
...
     if (message.length > MAX_MESSAGE_LENGTH) {
       throw new Error(`Discord error: Message size exceeded (2000 characters)`)
     }
-    const channelId =
-      channel === 'PUBLIC' ? this.publicChannelId : this.internalChannelId
-
-    return this.send(message, channelId)
+    if (channel === 'PUBLIC' && this.publicChannelId) {
+      return this.send(message, this.publicChannelId)
+    }
```
Comment (michalsidzej) [inline packages/backend/src/peripherals/discord/DiscordClient.ts:26]: Maybe log about the skip of the public channel? We don't want to forget adding publicChannelId in prod

Comment (sz-piotr) [reply packages/backend/src/peripherals/discord/DiscordClient.ts:26]: In production the `config.production.ts` enforces having `process.env.PUBLIC_DISCORD_CHANNEL_ID`

**Review by michalsidzej — APPROVED** [review]

**Pull request merged**